AzureCalling/External/Calling/CallingContext.swift

```diff
@@ -149,8 +148,10 @@ class CallingContext: NSObject {
             self.call?.startVideo(stream: localVideoStream) { (error) in
                 if error != nil {
                     print("ERROR: Local video failed to start. \(error!)")
+                    completionHandler(nil)
```

Comment:
Do we need this? We pass `completionHandler(self.localVideoStream)` before the CallingSDK callback closure is being called.  
This error apply to `calling SDK`'s failure in transporting local video stream, but locally we have the `localVideoStream`. If the callingSDK failed to pass the local video stream to SDK, do we want to display the self local video to the user.  
Same question apply to the `isCameraPreferredOn` movement. Do we want to trigger the flag only when the `callingSDK` successfully pass our local video?
@waynemo, what's your thought

Reply:
I think the intent of the flag is to indicate the preferred on/off state of the camera so that the UI can show the preferred selection. For example:

- If isCameraPreferredOn=false, the toggle button should show the "camera off" icon, and the local participant view would display the avatar image instead of the RendererView.
- If isCameraPreferredOn=true, the toggle button should show the "camera on" icon, and the local participant view would display the RendererView instead of the avatar image. If the localVideoStream is available, I think we should still be showing the local video since it is not the issue. 

The main concern would be how to indicate that the local video is not actually being sent. Let's discuss more offline.